### ISSUE TITLE
FocusZone: Cannot move to item prior to readOnly input with value

### ISSUE THREAD

**Issue by nkalpana** [OfficeDev/office-ui-fabric-react#11425]
### Environment Information

- **Package version(s)**: latest version (used by Export to CodePen option from controls playground)
- **Browser and OS versions**: Chrome + Narrator + On Screen keyboad

### Describe the issue:

I have a ShimmeredDetaislList with custom column renderers. Every column rendering method renders a Stack with a TextField and and a Copy Icon. 

Steps:
1) Use the On screen keyboard, or arrow keys to land focus on the first row, of the table
2) Now hit right arrow key. Focus moves to the text field in the first row first column, as expected
3) Hit right arrow key again. Focus moves to the text field in the first row second column, instead of stopping on the CopyIcon of the first column
4) Hit right arrow again, the focus does not move out of the second column's text field into the second columns CopyIcon
5) Hit left arrow now. The focus does not move out of the second column's text field back into the first column's text field or CopyIcon.

We received an A11y bug from the A11y team with the expectation that arrow key navigation should work left to right and backwards as well, within one row of a DetailsList.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/nkalpana/pen/eYmZEvm

#### Actual behavior:
Arrow key navigation focus works only one way, left to right, and stops at the right most column. Does not move back to the left columns if the user uses left key after getting to the last column.

We received an A11y bug from the A11y team with the expectation that arrow key navigation should work left to right and backwards as well, within one row of a DetailsList.

#### Expected behavior:
Arrow key navigation focus works only one way, left to right. Not right to left within the same row.

**Comment by micahgodbolt** [comment]
The issue is that read only TextFields are eating all of the left arrow key presses.

**Comment by nkalpana** [comment]
Thanks for triaging this Micah. Is there a duplicate issue I can refer to ?

**Comment by micahgodbolt** [comment]
One more note, this happens with normal inputs as well.

```jsx
<input onKeyDown={ev => console.log(ev)} readOnly defaultValue="I am read-only" />
```

event fires, but readOnly breaks FocusZone moving to prior item.

**Comment by micahgodbolt** [comment]
Lastly, got a reduced test case repro

https://codepen.io/micahgodbolt/pen/RwNaXrg?editors=0010

Note that this only happens for readOnly inputs with values, not empty or placeholders.

**Comment by nkalpana** [comment]
Thanks. So this is a genuine bug with the TextField control right ?

**Comment by micahgodbolt** [comment]
Bug in FocusZone :)
